tests/test_skeleton.py

```diff
@@ -0,0 +1,158 @@
+import os
```

Comment:
Have you tried starting this file with line
```py
# pylint: disable=W,C,R
```
?

Reply:
@shammamah, @mkcor : I tried and it worked (I use pylint on it locally and then added the line `# pylint: disable=E,W,C,R`)